easybuild/easyconfigs/x/XML-Compile/XML-Compile-1.63-GCCcore-11.2.0.eb

```
@@ -0,0 +1,61 @@
+easyblock = 'Bundle'
+
+name = 'XML-Compile'
+version = '1.63'
+
+homepage = 'https://metacpan.org/pod/XML::Compile'
+description = "Perl module for compilation based XML processing"
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+builddependencies = [
+    ('pkgconf', '1.8.0'),
+    ('binutils', '2.37'),
+]
+
+dependencies = [
+    ('Perl', '5.34.0'),
+    ('XML-LibXML', '2.0207'),
+]
+
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perl -e 'require %(ext_name)s'", '')
+
+exts_list = [
+    ('XML::LibXML::Simple', '1.01', {
+        'source_tmpl': 'XML-LibXML-Simple-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MARKOV/'],
+        'checksums': ['cd98c8104b70d7672bfa26b4513b78adf2b4b9220e586aa8beb1a508500365a6'],
+    }),
+    ('XML::Compile', version, {
+        'source_tmpl': 'XML-Compile-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MARKOV/'],
+        'checksums': ['4b0871ef4a70bff37266d531bebcd1d065b109e8f5c5e996f87189a9f92d595f'],
+    }),
+    ('XML::Compile::Cache', '1.06', {
+        'source_tmpl': 'XML-Compile-Cache-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MARKOV/'],
+        'checksums': ['591b136bd92842c81a5176082503f47df6d5cc4d8e0d78953ef1557f747038a0'],
+    }),
+    ('XML::Compile::SOAP', '3.27', {
+        'source_tmpl': 'XML-Compile-SOAP-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MARKOV/'],
+        'checksums': ['d656632fdc58665cbe62ede9c35d1f5a871917a79eba11033afd2041152866a8'],
+    }),
+    ('XML::Compile::WSDL11', '3.08', {
+        'source_tmpl': 'XML-Compile-WSDL11-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MARKOV/'],
+        'checksums': ['dd687ccf5083fe98fce1dd18540e1d0175042437a986e33eec105eca248f8d42'],
+    }),
+]
+
+modextrapaths = {
+    'PERL5LIB': 'lib/perl5/site_perl/%(perlver)s/',
+}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/perl5/site_perl/%(perlver)s/%(arch)s-linux-thread-multi/auto/XML/Compile'],
```

I'd probably check:
```suggestion
    'dirs': ['lib/perl5/site_perl/%(perlver)s/XML/Compile'],
```
which is where the `.pm` files are installed